test/services/supported_ssg_updater_test.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# A class to test the SupportedSsgUpdater service
+class SupportedSsgUpdaterTest < ActiveSupport::TestCase
+  context 'fallback file' do
+    setup do
+      @supported_file = ::Tempfile.new('suported_ssg_file')
+      @supported_file.write(File.read(SsgConfigDownloader::DS_FILE_PATH))
+      @supported_file.rewind
+
+      @fallback_file = ::Tempfile.new('fallback_ssg_file')
+      @fallback_file.write(File.read(SsgConfigDownloader::DS_FALLBACK_PATH))
+      @fallback_file.rewind
```

Comment:
You should not write the contents of the fallback file into the target. In case of testing it should be empty. Or in specific cases can have some bogus data and should be overridden.